masterlist.yaml

```diff
@@ -20453,16 +20453,29 @@ plugins:
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/33698/'
         name: 'DLC Frostcrag Cobl Addons'
+    msg:
+      - <<: *alreadyInX
+        subs: [ '[Frostcrag Spire Coblized](https://www.nexusmods.com/oblivion/mods/52608/)' ]
+        condition: 'active("DLCFrostcragCoblAlchemyTable.esp")'
     clean:
       - crc: 0xBF0955A6
         util: 'TES4Edit v4.0.4'
 
   - name: 'DLCFrostcragCoblAlchemyTable.esp'
     url:
       - link: 'https://www.nexusmods.com/oblivion/mods/52608/'
-        name: 'Frostcrag Spire Cobl Alchemy Station'
+        name: 'Frostcrag Spire Coblized'
+    msg:
+      - <<: *obsolete
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", <)'
+      - <<: *includesX
+        subs: [ 'DLC Frostcrag Cobl Addons' ]
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=) and active("DLCFrostcragCoblAddons.esp")'
+      - <<: *includesX
+        subs: [ 'Frostcrag Alchemy Sorter' ]
+        condition: 'version("DLCFrostcragCoblAlchemyTable.esp", "2.0", >=) and active("FrostcragAlchemySorter.esp")'
```

Comment:
```suggestion
```

These are redundant with the `alreadyInX` messages which is preferred in this context.